scripts/files-touched-check.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import os
+import sys
+import subprocess
+
+travis_commit_range = os.getenv('TRAVIS_COMMIT_RANGE')
+if not travis_commit_range:
+    print("Travis commit range is empty, exiting...")
+    sys.exit(0)
+
+try:
+    result = subprocess.check_output(['git', 'diff', '--no-commit-id', '--name-status', '-r', travis_commit_range])
+except Exception as e:
+    print(e.output)
+    raise e
+
+files_added = result.decode('utf-8').splitlines()
+print(files_added)
+subdir_name = ""
+    
+for file_added in files_added:
+    file_added = file_added.split()
```

Comment:
this can get confused if there are files with whitespace in the name - please use `file.added.split(maxsplit=1)` to make sure you only split the glyph from the filename